src/inject.js

```diff
@@ -55,6 +56,49 @@ const stringHandlers = {
             return val;
         }
     },
+
+    // With animationName we look for an object that contains keyframes and
+    // inject them as an `@keyframes` block, returning a uniquely generated
+    // name. The keyframes object should look like
+    //  animationName: {
+    //    from: {
+    //      left: 0,
+    //      top: 0,
+    //    },
+    //    '50%': {
+    //      left: 15,
+    //      top: 5,
+    //    },
+    //    to: {
+    //      left: 20,
+    //      top: 20,
+    //    }
+    //  }
+    // TODO(emily): figure out a way to call this `animation` instead of
+    // `animationName`.
+    animationName: (val) => {
+        if (typeof val === "object") {
```

Comment:
To tighten up the code a bit you could do:

```
if (typeof val !== "object") {
    return val;
}
```

and then just have the rest of the code (helps to save on an extra indentation).